donkeycar/parts/keras.py

```diff
@@ -152,134 +130,62 @@ def default_categorical():
     x = Dropout(.1)(x)  # Randomly drop out (turn off) 10% of the neurons (Prevent overfitting)
     x = Dense(50, activation='relu')(x)  # Classify the data into 50 features, make all negatives 0
     x = Dropout(.1)(x)  # Randomly drop out 10% of the neurons (Prevent overfitting)
-    # categorical output of the angle
-    angle_out = Dense(15, activation='softmax', name='angle_out')(
-        x)  # Connect every input with every output and output 15 hidden units. Use Softmax to give percentage. 15 categories and find best one based off percentage 0.0-1.0
 
-    # continous output of throttle
-    #throttle_out = Dense(1, activation='relu', name='throttle_out')(x)  # Reduce to 1 number, Positive number only
-    throttle_out = Dense(15, activation='softmax', name='throttle_out')(x)
+    angle_out = Dense(units=1, activation='linear', name='angle_out')(x)
+    throttle_out = Dense(units=1, activation='linear', name='throttle_out')(x)
```

Comment:
👍 Both angle and throttle make sense to be linear.

Reply:
I may be completely lost as I'm only looking at part of the code, but would there be sense in using tanh instead for angle_out & throttle_out activations, so that values very close to -1 or 1 (and values beyond) would be constricted to the range [-1, 1]?

Reply:
(Assuming we are still talking about continuous output and there's still a risk that the values overshoot)

Reply:
Noted that tanh. Have to try later